src/config/theme.js

```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
   ...bodhiTheme.overrides,
   MuiButton: {
     root: {
+      fontSize: xSmallText,
```

Comment:
should be 1rem, smallText

Reply:
It seems to be no `sizeMedium` property, so I overrode styles in `root` property.
Medium size button seems to be 14px.

Reply:
you are right, medium is the default size so root seems like the right spot. and for the font size i was mistaking the line height as the font size... sorry